adapters/pangle/pangle_test.go

```diff
@@ -0,0 +1,22 @@
+package pangle
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	conf := config.Adapter{
+		Endpoint:         "https://pangle.io/api/get_ads",
+		ExtraAdapterInfo: "{\"token\": \"123\"}",
```

Comment:
The adapter doesn't use the `ExtraAdapterInfo` field. Is this left over from an earlier design?

Reply:
yes it is. thanks for pointing that out